registration-processor/init/registration-processor-registration-status-service/src/main/java/io/mosip/registration/processor/status/api/controller/handler/RegistrationStatusExceptionHandler.java

```diff
@@ -1,165 +1,165 @@
-		regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.APPLICATIONID.toString(),
-				PlatformErrorMessages.RPR_SYS_BAD_GATEWAY.getCode(), "langCode must be of 3 characters");
-		return buildRegStatusExceptionResponse((Exception) ex);
-	}
-
-	@ExceptionHandler(DataIntegrityViolationException.class)
-	public ResponseEntity<Object> dataExceptionHandler(final DataIntegrityViolationException e, WebRequest request) {
-
-		regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.APPLICATIONID.toString(),
-				"RPR-DBE-001 Data integrity violation exception", e.getMessage());
-		return buildRegStatusExceptionResponse((Exception) e);
-	}
-
-	@ExceptionHandler(RegStatusAppException.class)
-	protected ResponseEntity<Object> handleRegStatusException(RegStatusAppException e, WebRequest request) {
-		return buildRegStatusExceptionResponse((Exception) e);
-
-	}
-
-	@ExceptionHandler(InvalidTokenException.class)
-	protected ResponseEntity<Object> handleInvalidTokenException(InvalidTokenException e, WebRequest request) {
-		return buildRegStatusExceptionResponse((Exception) e);
-
-	}
-
-	private ResponseEntity<Object> buildRegStatusExceptionResponse(Exception ex) {
-
-		RegStatusResponseDTO response = new RegStatusResponseDTO();
-		Throwable e = ex;
-
-		if (Objects.isNull(response.getId())) {
-			response.setId(env.getProperty(REG_STATUS_SERVICE_ID));
-		}
-		if (e instanceof BaseCheckedException)
-
-		{
-			List<String> errorCodes = ((BaseCheckedException) e).getCodes();
-			List<String> errorTexts = ((BaseCheckedException) e).getErrorTexts();
-
-			List<ErrorDTO> errors = errorTexts.parallelStream()
-					.map(errMsg -> new ErrorDTO(errorCodes.get(errorTexts.indexOf(errMsg)), errMsg)).distinct()
-					.collect(Collectors.toList());
-
-			response.setErrors(errors);
-		}
-		if (e instanceof BaseUncheckedException) {
-			List<String> errorCodes = ((BaseUncheckedException) e).getCodes();
-			List<String> errorTexts = ((BaseUncheckedException) e).getErrorTexts();
-
-			List<ErrorDTO> errors = errorTexts.parallelStream()
-					.map(errMsg -> new ErrorDTO(errorCodes.get(errorTexts.indexOf(errMsg)), errMsg)).distinct()
-					.collect(Collectors.toList());
-
-			response.setErrors(errors);
-		}
-
-		response.setResponsetime(DateUtils.getUTCCurrentDateTimeString(env.getProperty(DATETIME_PATTERN)));
-		response.setVersion(env.getProperty(REG_STATUS_APPLICATION_VERSION));
-		response.setResponse(null);
-		Gson gson = new GsonBuilder().serializeNulls().create();
-
-		if (isEnabled) {
-			HttpHeaders headers = new HttpHeaders();
-			headers.add(RESPONSE_SIGNATURE, digitalSignatureUtility.getDigitalSignature(gson.toJson(response)));
-			return ResponseEntity.ok().headers(headers).body(gson.toJson(response));
-		}
-		return ResponseEntity.status(HttpStatus.OK).body(response);
-
-	}
-
-}
+package io.mosip.registration.processor.status.api.controller.handler;

```

Comment:
If there is no change in this file, please don't commit this

Reply:
This is already committed some space changes only